packages/react-scripts/bin/react-scripts.js

```diff
@@ -43,6 +43,19 @@ switch (script) {
     process.exit(result.status);
     break;
   }
+  case 'lint': {
+    spawn.sync(
+      'node',
+      [
+        'node_modules/eslint/bin/eslint.js',
```

Comment:
I would prefer that we use `require.resolve` here rather than hardcode `node_modules` path.